pom.xml

```
@@ -19,7 +19,7 @@
 		<kotlinx-coroutines.version>1.6.2</kotlinx-coroutines.version>
 		<spring-cloud-azure.version>4.0.0</spring-cloud-azure.version>
 		<jacoco.version>0.8.8</jacoco.version>
-		<pagopa-ecommerce-commons.version>1.19.0</pagopa-ecommerce-commons.version>
+		<pagopa-ecommerce-commons.version>1.19.1</pagopa-ecommerce-commons.version>
```

```suggestion
		<pagopa-ecommerce-commons.version>1.19.2</pagopa-ecommerce-commons.version>
```
TODO to be replaced after dependent commons library PR merge